lib/Doctrine/DBAL/Migrations/Configuration/Connection/Loader/ArrayConnectionConfigurationLoader.php

```diff
@@ -8,6 +8,7 @@
 
 class ArrayConnectionConfigurationLoader implements ConnectionLoaderInterface
 {
+    /** @var string */
     private $filename;
 
     public function __construct($filename)
```

Comment:
Too bad we can't require the typehint sniffs because they'd introduce BC breaks. I'll create an issue to make it optional so we can require parameter documentation without introducing BC breaks.

Reply:
Just cross-referencing https://github.com/slevomat/coding-standard/issues/242 here - the sniff needs to be split according to the maintainers there. Maybe if it hurts me too much I'll give that a shot.

Reply:
for the constructor, you could typehint the string. It does not create a signature mismatch. And passing something else than a string would explode on the `file_exists` call already.